docs/manifest.json

```diff
@@ -1109,22 +1109,16 @@
 		"markdown_source": "../packages/components/src/navigator/navigator-button/README.md",
```

Comment:
I see we continue calling these for example `NavigatorButton`, but they're actually exposed as `Navigator.Button`? Should this be reflected in the manifest? Should we just leave the single `Navigator` and have all READMEs in it?

Reply:
I think the name is auto-generated from the filename or something else. I like the idea of moving everything under one README, though. We currently have some compound components having one unique README, and other compound components with one README per sub-component. It would be good if we agreed on an approach going forward — @WordPress/gutenberg-components , any preferences?

Reply:
I merged all READMEs into one: https://github.com/WordPress/gutenberg/pull/64613/commits/5e33f26ba86f2308b331a0b7af60feb931836727 and https://github.com/WordPress/gutenberg/pull/64613/commits/0cff1065813626fde5a2e14f444e69dbe39c6bfd.

I think I like keeping one README more, but happy to hear other's thoughts before merging the PR.

Reply:
> I think I like keeping one README more

No strong opinion. I might have an updated opinion after I attempt auto-generating the READMEs 🙈

Reply:
I prefer a single README too - primarily because it's a better developer experience. It's way easier to understand the entire component family, rather than having to jump between multiple READMEs and figure out the puzzle on your own.

Reply:
Sounds good — I guess we can go with the one README for now and tweak later if necessary, together with the rest of the components that will need similar changes.